src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -368,6 +387,102 @@ static MvcCoreLoggerExtensions()
+               10,
+               "Could not bind to model with name '{ModelName}' and type '{ModelType}' as there were no public settable properties on it.");
+
+            _cannotBindToComplexType = LoggerMessage.Define<Type>(
+               LogLevel.Debug,
+               11,
+               "Could not bind to model of type '{ModelType}' as there were no values for any of the properties on the request to bind from.");
+
+            _cannotBindToFilesCollectionDueToInvalidContentType = LoggerMessage.Define<string, Type>(
+               LogLevel.Debug,
+               12,
+               "Could not bind to model with name '{ModelName}' having the model type '{ModelType}' as the request did not have content type of either 'application/x-www-form-urlencoded' or 'multipart/form-data'.");
+
+            _cannotCreateHeaderModelBinder = LoggerMessage.Define<Type>(
+               LogLevel.Debug,
+               13,
+               "Could not create binder to bind model with type '{ModelType}' as this binder only supports 'System.String' type or collection of 'System.String' type.");
+
+            _noFilesFoundOnTheRequest = LoggerMessage.Define(
+                LogLevel.Debug,
+                14,
+                "No files found on the request to bind the model to.");
+
+            _tryingToBindParameter = LoggerMessage.Define<string, string, Type>(
+                LogLevel.Debug,
+                15,
+                "Trying to bind {ParameterOrProperty} '{ParameterOrPropertyName}' of type '{ModelType}' ...");
+
+            _doneTryingToBindParameter = LoggerMessage.Define<string, string, Type>(
+                LogLevel.Debug,
+                16,
+                "Done trying to bind {ParameterOrProperty} '{ParameterOrPropertyName}' of type '{ModelType}'.");
+
+            _tryingToBindModel = LoggerMessage.Define<Type>(
+                LogLevel.Debug,
+                17,
+                "Trying to bind model of type '{ModelType}' ...");
+
+            _doneTryingToBindModel = LoggerMessage.Define<Type>(
+                LogLevel.Debug,
+                18,
+                "Done trying to bind model of type '{ModelType}'.");
+
+            _tryingToValidateParameter = LoggerMessage.Define<string, string, Type>(
+                LogLevel.Debug,
+                19,
+                "Trying to validate the bound {ParameterOrProperty} '{ParameterOrPropertyName}' of type '{ModelType}' ...");
+
+            _doneTryingToValidateParameter = LoggerMessage.Define<string, string, Type>(
+                LogLevel.Debug,
+                20,
+                "Done trying to validate the bound {ParameterOrProperty} '{ParameterOrPropertyName}' of type '{ModelType}'.");
+
+            _noNonIndexBasedFormatFoundForCollection = LoggerMessage.Define<string, string>(
+                LogLevel.Debug,
+                21,
+                "Could not bind to collection using the format like {ModelName}=value1&{ModelName}=value2 etc.");
```

Comment:
"the format like" -> "a format like" and drop "etc" (it and "like" imply the same thing here.)